src/rest/endpoints.ts

```diff
@@ -1366,9 +1366,9 @@ The `emoji` must be [URL Encoded](https://en.wikipedia.org/wiki/Percent-encoding
   }
 
   /**
-   * Creates a new public thread from an existing message. Returns a channel on success, and a 400 BAD REQUEST on invalid parameters. Fires a Thread Create Gateway event.
+   * Creates a new thread from an existing message. Returns a channel on success, and a 400 BAD REQUEST on invalid parameters. Fires a Thread Create Gateway event.
    */
-  async startPublicThread(
+  async startPublicThreadFromMessage(
```

Comment:
Breaking change?

Reply:
oh right totally forgot to add back compatibility